### ISSUE TITLE
Accessibility issues on city template

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2354]
## Source

LF

## Describe the bug

There are a few minor a11y errors in city template that should be cleaned up

## To reproduce
1. Go to demo.proudcity.com
2. Review page using WAVE
3. See errors and alerts.

## Expected behavior

No errors and ideally no alerts.

## Screenshots

![Screenshot 2023-09-14 at 1 24 17 PM](https://github.com/proudcity/wp-proudcity/assets/1569463/6ae433bf-5543-45d6-aa1e-56c6e86c838b)

## QA

Notes:

* 

Links:

*

**Comment by kevindherman** [comment]
@curtismchale any luck identifying what the items are being flagged here? cc @lukefretwell

**Comment by curtismchale** [comment]
**5 Form Errors**
The first entry applies only to the Google Translate form that allows you to rate your feedback on a translation. 

![CleanShot 2023-09-20 at 06 42 38](https://github.com/proudcity/wp-proudcity/assets/74561/db0cf9ea-823e-4607-9c27-a17cc7fff17c)

https://github.com/proudcity/wp-proudcity/issues/2371

**Contrast Error 1**

This is the action toolbar with a class of `.sr-only`. See: https://github.com/proudcity/wp-proudcity/issues/2362

**Contrast Error 2**

Primary menu label is low contrast: https://github.com/proudcity/wp-proudcity/issues/2363

**Orphaned Form Labels**
- https://github.com/proudcity/wp-proudcity/issues/2364
- https://github.com/proudcity/wp-proudcity/issues/2365

**Rendundant links**
- https://github.com/proudcity/wp-proudcity/issues/2366
- https://github.com/proudcity/wp-proudcity/issues/2367
- https://github.com/proudcity/wp-proudcity/issues/2368
- https://github.com/proudcity/wp-proudcity/issues/2369
- https://github.com/proudcity/wp-proudcity/issues/2370
- https://github.com/proudcity/wp-proudcity/issues/2372
- https://github.com/proudcity/wp-proudcity/issues/2373
- https://github.com/proudcity/wp-proudcity/issues/2374
- https://github.com/proudcity/wp-proudcity/issues/2375

**Small Text**
- https://github.com/proudcity/wp-proudcity/issues/2376


I stil have some more to go through so this will update once I get kids to school.
